tensorflow/contrib/lite/python/lite.py

```diff
@@ -346,7 +346,8 @@ def from_keras_model_file(cls,
                             model_file,
                             input_arrays=None,
                             input_shapes=None,
-                            output_arrays=None):
+                            output_arrays=None,
```

Comment:
1. Add a description of the argument `custom_objects` to the docstring (follow the format of the other arguments).

2. Add a test function that uses `custom_objects` to the `FromKerasFile` class in `lite_test.py`. Make sure all of the tests in `FromKerasFile` in `lite_test.py` run. Feel free to use `--test_filter=FromKerasFile` or temporarily comment out the tests `testTFLiteGraphDef` and `testTFLiteGraphDefMissingShape` when you are testing locally since those tests currently don't work in open source.

Reply:
:+1: Give me a bit of time to work on it.

Reply:
@gargn @rmlarsen Done!